data/json/items/comestibles/wheat.json

```diff
@@ -20,6 +20,27 @@
     "vitamins": [ [ "calcium", 4 ], [ "iron", 40 ] ],
     "fun": -10
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "grain_boiled",
+    "looks_like": "buckwheat_cooked",
+    "name": { "str_sp": "boiled grain" },
+    "spoils_in": "10 days",
+    "weight": "260 g",
+    "color": "brown",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "healthy": 1,
+    "calories": 705,
```

Comment:
Negative morale for this one as well, I think.

Reply:
It's already negative; "fun" entry at the bottom of this entry.